.scribe/SCRIBE.md

```diff
@@ -0,0 +1,55 @@
+#### Working with Scribe API Documentation
+
+Access the documentation:
+
+- With Docker:
+    - `learnhub.test:8000/docs`
+- Without Docker
+    - `localhost:8000/docs`
+
+In order to generate documentation for APIs use: `php artisan scribe:generate`
+
+#### Working with Socialite
```

Comment:
This section does not have anything to do with Scribe. You can move into a Socialite markdown file.

Reply:
Ok, but where should that file be located? Same folder? Please be more specific.

Reply:
The root of the project should be good.

Reply:
Same goes for the Scribe docs

Reply:
Done, you can resolve this. 
I don't understand what Scribe docs is.
Also I think "learnhub.test:8080" doesn't even work via Docker.

Reply:
It works if you have the APP_PORT and APP_SERVICE set up in your .env file.

It doesn't work right now because there were some changes to the docker-compose.yaml file.

Will fix it soon.